meteor-app/imports/api/parts/methods.js

```diff
@@ -1,111 +1,117 @@
-import { Meteor } from 'meteor/meteor'
-import Parts from '/imports/api/parts/schema'
-import log from '/imports/lib/server/log'
-import XLSX from 'xlsx'
-const debug = require('debug')('b2b:parts')
+import { Meteor } from "meteor/meteor";
+import Parts from "/imports/api/parts/schema";
+import log from "/imports/lib/server/log";
+import XLSX from "xlsx";
+const debug = require("debug")("b2b:parts");
 
-function calcRetail(price){
+function calcRetail(price) {
   if (price <= 6000) {
-    const retailPrice = parseInt(price * 100, 10) * 2
-    return retailPrice
-  }
-  else if (price > 6000 && price <= 10000) {
-    const retailPrice = parseInt(price * 100, 10) * 1.5
-    return retailPrice
-  }
-  else if (price > 10000) {
-    const retailPrice = parseInt(price * 100, 10) * 1.3
-    return retailPrice
-  }
-  else {
-    return console.error('Error, not a number')
+    try {
+      const retailPrice = parseInt(price, 10) * 100 * 2;
+      return retailPrice;
+    } catch (e) {
+      console.error(e);
```

Comment:
This is an improvement, but all it's going to do is print something like 'Invalid number', which won't help you find which record caused the problem.